BiLiRoku/About.xaml

```diff
@@ -18,14 +18,17 @@
     </Window.CommandBindings>
     <Grid>
         <Grid.ColumnDefinitions>
-            <ColumnDefinition Width="129*"/>
-            <ColumnDefinition Width="335*"/>
```

Comment:
I don't know what are you dissatisfied with ratio 129: 335, but this is implemented by design.